flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -39,8 +39,7 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
         infos["integrations"] = integrations
       }
 
-      // TODO get the sdkVersion from SDK
-      infos["package"] = ["version": "6.0.9", "sdk_name": "cocoapods:sentry-cocoa"]
+      infos["package"] = ["version": "\(Options().sdkInfo.version)", "sdk_name": "cocoapods:sentry-cocoa"]
```

Comment:
can we do:

`sentryOptions.sdkInfo.version`?
theres already an instance of it

Reply:
`sentryOptions` is an optional member of `SwiftSentryFlutterPlugin`. However, it should always have a value when this is called, or is there a code path that would call this before `initNativeSdk`?

Reply:
integrations are executed in order, so `initNativeSdk` is always called before, but I'd definitely check if `sentryOptions` is not null for the sake of safety, if `sentryOptions. sdkInfo` is not available, I'd not set the `infos["package"]` at all.